managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -2355,7 +2361,8 @@ public void operationFailed(ManagedLedgerException e) {
             persistPositionMetaStore(-1, position, properties, new MetaStoreCallback<Void>() {
                 @Override
                 public void operationComplete(Void result, Stat stat) {
-                    log.info("[{}][{}] Closed cursor at md-position={}", ledger.getName(), name, markDeletePosition);
+                    log.info("[{}][{}] Closed cursor at md-position={} individualDeletedMessagesSize={}",
+                            ledger.getName(), name, markDeletePosition, individualDeletedMessages.size());
```

Comment:
Log `individualDeletedMessages.size()` seems useless.

Reply:
Well, it looks like the log here is not accurate, I will add a log in front